src/js/utils.js

```diff
@@ -187,4 +187,39 @@ export function formatNumericalValue(number, formatting, method) {
     }
 
     return fn(number);
-}
\ No newline at end of file
+}
+
+function isNull(x) {
+    return x === null;
```

Comment:
these checks are much better than the `type of... `

Reply:
apart from undefined - which one would you change? 

null is of type object so it needs == null